Adding storage gloss terms (#335)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

// AMQ: Added "In Red Hat AMQ, a target is"
[[target]]
==== image:images/yes.png[yes] target (noun)

> *Description*: 1) Generically, synonymous with destination, a _target_ is a file, device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. This is usually a queue or topic.

Reviewer: ```suggestion
*Description*: 1) Generically, a _target_ is synonymous with destination. A _target_ can be a file, a device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. The _target_ is usually a queue or topic.
```
I would split this entry into a bulleted list, as I suggested for previous similar entries.

> *Description*: 1) Generically, a _target_ is synonymous with destination. A _target_ can be a file, a device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. The _target_ is usually a queue or topic.

Reviewer 2: ```suggestion
*Description*: Generically, a _target_ is synonymous with destination. A _target_ can be a file, a device or any type of location to which data is moved or copied. In Red Hat AMQ, a _target_ is a message's destination. The _target_ is usually a queue or topic.
```

[discrete]
[[target]]
==== image:images/yes.png[yes] target (noun)

> *Description*: 1) Generically, synonymous with destination, a target is a file, device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. This is usually a queue or topic.

Reviewer 3: ```suggestion
*Description*: 1) Generically, synonymous with destination, a _target_ is a file, device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. This is usually a queue or topic.
```

*See also*:

[[thickly-provisioned]]
==== image:images/yes.png[yes] thickly provisioned (adjective)

> *Description*: Disk storage that is pre-allocated so that the total amount of virtual disk storage capacity exists on the physical storage when a virtual disk is created.

Reviewer: ```suggestion
*Description*: _Thickly provisioned_ is used with reference to disk storage that is pre-allocated so that the total amount of virtual disk storage capacity exists on the physical storage when a virtual disk is created.
```

*See also*:

[[total-capacity]]
==== image:images/yes.png[yes] total capacity (noun)

> *Description*: The complete amount of used and unused space available on a storage device.

Reviewer: ```suggestion
*Description*: _Total capacity_ is the complete amount of used and unused storage space available on a storage device.
```

*See also*:

[[translator]]
==== image:images/yes.png[yes] translator (noun)

> *Description*: In Red Hat Gluster Storage, a software module that converts user requests into a storage operation. Multiple translators may interact with the data in the request, but translators always work in a specific order, with data passed from one to another as appropriate, depending on the gluster volume configuration.

Reviewer: ```suggestion
*Description*: In Red Hat Gluster Storage, a _translator_ is a software module that converts user requests into a storage operation. Multiple translators may interact with the data in the request, but translators always work in a specific order, with data passed from one to another as appropriate, depending on the Gluster volume configuration.
```
